### PR TITLE
(feat) 006: add third-party libraries agenda

### PR DESCRIPTION
**Pull request by Liza-Veis**

### CODE REVIEW

File: third-party-libs/index.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+## Table of contents
+
+1. Finding libraries
+2. Determining library compatibility
+3. Installing and linking libraries
+   1. Android
+   2. IOS
+4. Adding icons
+5. Google Maps integration
+6. Stripe integration
```
Comment (iralitv) [inline third-party-libs/index.md:12]: Does it make sense to add topic about communication between RN and native modules here?

Comment (Liza-Veis) [reply third-party-libs/index.md:12]: will cover this in [React Native Advanced module](https://github.com/rolling-scopes-school/react-native-course/issues/57)

**Review by Karina-Trosko — APPROVED** [review]

File: third-party-libs/index.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-- ToDo
-- ToDo
+## Table of contents
+
+1. Finding libraries
+2. Determining library compatibility
+3. Installing and linking libraries
+   1. Android
+   2. IOS
+4. Adding icons
```
Comment (Dzmitry-Klokau) [inline third-party-libs/index.md:10]: what do you think about "Adding custom fonts"?

Comment (adenyx) [reply third-party-libs/index.md:10]: I suppose this is very important topic for beginners and sometimes it's hard to implement for them.

**Review by adenyx — APPROVED** [review]

**Pull request merged**